central/processlisteningonport/datastore/singletone.go

```diff
@@ -0,0 +1,26 @@
+package datastore
+
+import (
+	"github.com/stackrox/rox/central/globaldb"
+	processIndicatorStore "github.com/stackrox/rox/central/processindicator/datastore"
+	"github.com/stackrox/rox/central/processlisteningonport/store/postgres"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	once sync.Once
+
+	store DataStore
+)
+
+func initialize() {
+	plopStorage := postgres.New(globaldb.GetPostgres())
```

Comment:
nit:  i would differentiate the names to make it clear that one is the store and one references the datastore